docs/BUILDING_MACOS.md

```diff
@@ -11,13 +11,13 @@ For macOS, use Homebrew to install the following dependencies:
 * python3
 * libpng
 * bash
-* clang-format 11
+* clang-format-14
 
 You can install them with the following commands:
 
 ```bash
 brew update
-brew install coreutils make python3 libpng bash clang-format@11
+brew install coreutils make python3 libpng bash clang-format@14
```

Comment:
Just noticed a problem, the homebrew page for clang-format 14 doesn't seem to exist
https://formulae.brew.sh/formula/clang-format@14

As a reference this is version 11: https://formulae.brew.sh/formula/clang-format@11